TeaDiary/Scripts/App/TeaListDisplayer.js

```diff
@@ -1,6 +1,7 @@
 ﻿function TeaListDisplayer(container) {
     var self = this;
     self.container = $(container);
+    var teaElementId = "displayedTeaElementNumber";
```

Comment:
1. Why did you add this?
2. Why isn't this a constant?

Reply:
1. To assign tea element and Id. And have it as sort of constant so it will be easier to operate
2. Moved to constant section